framework/src/media/demux/mpeg2ts/Section.cpp

```diff
@@ -0,0 +1,103 @@
+/******************************************************************
+ *
+ * Copyright 2019 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ******************************************************************/
+
+#include <stdio.h>
+#include <string.h>
+#include "Section.h"
+#include "Mpeg2TsTypes.h"
+#include "../../utils/MediaUtils.h"
+
+
+Section::Section(uint16_t u16Pid, uint8_t continuityCounter, uint8_t *pu8Data, uint16_t u16Size)
+	: m_section_length(0)
```

Comment:
`m_section_length` and `m_data` will be initialized in constructor.
I think this line can be removed, isn't it?

Reply:
fixed, thanks